src/clarity-angular/data/datagrid/providers/hideable-column.service.ts

```diff
@@ -192,9 +192,9 @@ export class HideableColumnService {
      * @returns HideableColumn
      *
      */
-    public getColumnById( id: string ): DatagridHideableColumn {
+    public getColumnById( id: string ): any { // Can't be DatagridHideableColumn b/c we don't always return something.
```

Comment:
Instead of declaring `any` here, would it make more sense to declare a union type?

```
public getColumnById( id: string ): void | DatagridHideableColumn {
    ...
}
```